app/assets/stylesheets/content/_attributes_key_value.sass

```diff
@@ -28,6 +28,8 @@
 
 .attributes-key-value
   @include grid-block($wrap: true)
+  overflow: visible
```

Comment:
@0xF013 IMO the overflow values for Foundation's grid mixins are not intuitive. I may look at moving these properties into a mixin, but probably not until #18329 is merged.